comps/dataprep/vdms/multimodal_langchain/Dockerfile

```diff
@@ -21,8 +21,8 @@ USER user
 
 COPY comps /home/user/comps
 
-RUN pip install --no-cache-dir --upgrade pip setuptools && \
-    if [ ${ARCH} = "cpu" ]; then pip install --no-cache-dir torch torchvision --index-url https://download.pytorch.org/whl/cpu; fi && \
+RUN pip install --no-cache-dir --upgrade pip setuptools  && \
+    if [ ${ARCH} = "cpu" ]; then pip install --no-cache-dir torch==2.4.0 torchvision --index-url https://download.pytorch.org/whl/cpu; fi && \
```

Comment:
why use a fixed version for torch?

Reply:
I fixed versions and checked.
Looks like PR #928 is merged which does same fix as this #922 . So we can close this.